tests/test_suites.py

```diff
@@ -124,3 +126,16 @@ def test_that_a_suite_will_consider_required_tests_when_passing(test_targets):
     suite = SuiteABC.from_target(target, required_tests)
     suite_status = suite.compute_status()
     assert suite_status == SuiteStatus.GREEN
+
+
+def test_that_status_is_computed_if_not_already_assigned(test_targets):
+    with patch.object(
+        SuiteABC, "compute_status", return_value=SuiteStatus.GREEN
```

Comment:
What is your rationale for patching this here?  If some code in `compute_status` is wrong, would that cause the wrong SuiteStatus to be returned?

Reply:
I wanted to be able to assert that the function was called, as it should when the status has not yet been computed and also be able to control which status to expect for this test.